docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -739,6 +739,29 @@ p2p-host="0.0.0.0"
 Specifies the host on which P2P listens.
 The default is 127.0.0.1.
 
+### p2p-interface
+
+```bash tab="Syntax"
+--p2p-interface=<HOST>
+```
+
+```bash tab="Command Line"
+# to listen on all interfaces
```

Comment:
Pretty sure this comment line should be removed - it's only relevant if the interface is 0.0.0.0 isn't it?

Reply:
Yep, copied and pasted, but forgot to remove this.